helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAccessor.java

```diff
@@ -75,8 +75,11 @@ public void testGetResources() throws IOException {
 
     Set<String> resources = OBJECT_MAPPER.readValue(idealStates,
         OBJECT_MAPPER.getTypeFactory().constructCollectionType(Set.class, String.class));
-    Assert.assertEquals(resources, _resourcesMap.get("TestCluster_0"), "Resources from response: "
-        + resources + " vs clusters actually: " + _resourcesMap.get("TestCluster_0"));
+    Assert.assertEquals(resources.size(), _resourcesMap.get("TestCluster_0").size(), "Sizes are not equal");
```

Comment:
Can you make the test error messages more consistent with what it was originally? For instance, In the original message they print the actual contents of resources.